clouddriver-ecs/src/test/groovy/com/netflix/spinnaker/clouddriver/ecs/provider/agent/TaskHealthCachingAgentSpec.groovy

```diff
@@ -114,6 +115,75 @@ class TaskHealthCachingAgentSpec extends Specification {
     taskHealth.getTaskId() == CommonCachingAgent.TASK_ID_1
   }
 
+  def 'should get a list of task health for tasks with multiple network bindings'() {
```

Comment:
If you merge from master (#3220 is now merged), you can reduce a lot of this copy-and-paste

Reply:
Oh excellent, didn't realized that had been merged. Will do.